en/releases/centreon-core.md

```diff
@@ -18,6 +18,37 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web release notes
 
+### 20.10.2
```

Comment:
Add a known bug section to warn about an issue related to macros not replaced in url built in Resources status: https://centreon.atlassian.net/browse/MON-6532 (not fixed yet)